src/components/AddPeopleAndRoles/ListPeopleAndRoles.vue

```diff
@@ -0,0 +1,253 @@
+                <v-list class="actions__more-actions">
+                  <v-list-item @click="confirmRemove(index)">
+                    <v-list-item-title><v-icon>mdi-delete</v-icon>Remove</v-list-item-title>
+                  </v-list-item>
+                </v-list>
+              </v-menu>
+            </span>
+          </div>
+        </v-col>
+      </v-row>
+    </div>
+  </v-card>
+</template>
+
+<script lang="ts">
+// Libraries
+import { Component, Prop, Mixins, Emit } from 'vue-property-decorator'
+
+// Components
+import BaseAddress from 'sbc-common-components/src/components/BaseAddress.vue'
+
+// Dialogs
+import { ConfirmRemoveDialog } from '@/components/dialogs'
+
+// Mixins
+import CommonMixin from '@/mixins/common-mixin'
+
+// Interfaces
+import { OrgPersonIF } from '@/interfaces'
+
+@Component({
+  components: {
+    BaseAddress,
+    ConfirmRemoveDialog
+  }
+})
+export default class ListPeopleAndRoles extends Mixins(CommonMixin) {
+  // Store Properties
+  @Prop({ default: '' })
+  private personList: Array<OrgPersonIF>
+
+  @Prop({ default: false })
+  private showErrorSummary: boolean
+
+  @Prop({ default: false })
+  private isSummary: boolean
+
+  // Local Properties
+  readonly tableHeaders: Array<string> = ['Name', 'Mailing Address', 'Delivery Address', 'Roles']
+  private dialog: boolean = false
+  private personId: number
+
+  /**
+   * Determine if Corporation/Firm or Person.
+   * @param filing The filing body which contains the name/title.
+   * @returns The appropriate Corporation or Person name.
+   */
+  private formatName (filing: any): string {
+    return filing.orgName ? filing.orgName
+      : `${filing.firstName} ${filing.middleName || ''} ${filing.lastName}`
+  }
+
+  /**
+   * Identify and confirm the removal of a person/org through dialog.
+   * @param index The active index which is subject to removal.
+   */
+  confirmRemove (index: number): void {
+    this.personId = index
+    this.dialog = true
+  }
+
+  /**
+   * Emit an index and event to the parent to handle removal.
+   * @param index The active index which is subject to removal.
+   */
+  @Emit('removePerson')
+  private emitRemovePerson (index: number): void {
+    this.dialog = false
+  }
+
+  /**
+   * Emit an index and event to the parent to handle editing.
+   * @param index The active index which is subject to change.
+   */
+  @Emit('editPerson')
+  private emitPersonInfo (index: number): void {}
+}
+</script>
+
+<style lang="scss" scoped>
+#people-roles {
+  margin-top: 1rem;
+}
+
+.people-roles-summary-header {
+  display: flex;
+  background-color: rgba(1, 51, 102, 0.15);
+  padding: 1.25rem;
+}
+
+.people-roles-invalid-message {
+  padding: 1.25rem;
+  font-weight: bold;
+  color: #1976d2;
+}
+
+.people-roles-header {
+  padding: .5rem 1.25rem .5rem 1.25rem;
+  font-size: 12px;
+  margin-top: 1rem;
+}
+
+.people-roles-content {
+  margin-top: .5rem;
+  padding: .5rem 1.25rem .5rem 1.25rem;
+  border-top: 1px solid #d1d1d1;
```

Comment:
Check for a named color (or color variable). This color is used in a couple of places. It would be better if it wasn't hard-coded.

Reply:
Good question. The designs don't actually translate to the specific colours in the themes. Same goes for the blue hex colour given in the design. I feel like i'm just guessing at which colour in the theme is closest..
Suggestions?

Reply:
Check this page for named colours and try to find a visual match. (I don't think we need exact RGB values.)
https://vuetifyjs.com/en/styles/colors/

Reply:
... And we should ask the designers to pick colours from the theme :)

Reply:
Agreed!